tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3009,7 +3009,9 @@ public function userGetsTheFollowingPropertiesOfFileInsideSpaceUsingTheWebdavApi
 		TableNode $propertiesTable
 	):void {
 		$this->setSpaceIDByName($user, $spaceName);
-		$this->webDavPropertiesContext->userGetsPropertiesOfFolder($user, $resourceName, $propertiesTable);
+		$response = $this->webDavPropertiesContext->getPropertiesOfFolder($user, $resourceName, $propertiesTable);
+		$this->featureContext->setResponse($response);
+		$this->featureContext->setResponseXmlObject(HttpRequestHelper::getResponseXml($response));
```

Comment:
this should not be necessary I guess. Could you try it?
```suggestion
```

Reply:
tests failed.
looks like some then steps check straight from the `responseXmlObject` and
`setResponse()` does not set the `responseXmlObject`

Reply:
Similar to https://github.com/owncloud/ocis/pull/7400/files. If you will implement it then I can merge this and rebase my PR, no problem